src/localizations/no_nb.json

```diff
@@ -109,12 +114,23 @@
     "search_hits": "treff på søket:",
     "missing_role_access": "Brukeren har ikke noen av de nødvendige rollene for å delegere tjenesten. (urn:altinn:rolecode:HADM, urn:altinn:location:garage), vil gi tilgang til å delegere den.",
     "missing_delegation_access": "Brukeren har ikke de nødvendige delegeringsrettighetene til å delegere tjenesten",
+    "missing_srr_right_access": "Brukeren har ikke de nødvendige SRR-rettighetene",
```

Comment:
It's because UX hadn't made texts for specific reasons of why something's not delegable and I had to make them on the go. Was thinking UX could set the correct texts in DS review. But I think I just change the texts to 'Brukeren' and then the same for the other texts and if design review complains we'll take the discussion if the texts should say 'Brukeren' or 'you'. What do you think? 'You' or 'Brukeren'?

Reply:
That's fair. We'll hear it from them soon enough if they have issues with anything.

As for what I think, well, since the rest of the page addresses the user directly (with 'you'), I think we should probably try to stick to that convention.

Reply:
Alright I change to 'you' instead. Although I thinknit sounds more unproffesional